app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  validates :name, presence: true, length: { maximum: 50 }
```

Comment:
move constant like 50, 255 to settings.yml file

Reply:
done